dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -15,65 +13,39 @@ namespace Microsoft.SemanticKernel.Orchestration;
 /// <summary>
 /// Semantic Kernel context.
 /// </summary>
-[DebuggerDisplay("{DebuggerDisplay,nq}")]
-public sealed class SKContext
+public abstract class SKContext
 {
-    /// <summary>
-    /// The culture currently associated with this context.
-    /// </summary>
-    private CultureInfo _culture;
-
     /// <summary>
     /// Print the processed input, aka the current data after any processing occurred.
     /// </summary>
     /// <returns>Processed input, aka result</returns>
-    public string Result => this.Variables.ToString();
+    public abstract string Result { get; }
 
     /// <summary>
     /// Whether an error occurred while executing functions in the pipeline.
     /// </summary>
-    public bool ErrorOccurred { get; private set; }
+    public abstract bool ErrorOccurred { get; }
 
     /// <summary>
     /// Error details.
     /// </summary>
-    public string LastErrorDescription { get; private set; } = string.Empty;
+    public abstract string LastErrorDescription { get; }
 
     /// <summary>
     /// When an error occurs, this is the most recent exception.
     /// </summary>
-    public Exception? LastException { get; private set; }
+    public abstract Exception? LastException { get; }
```

Comment:
No, this is just to minimize churn until new mechanism is in place. If we agree that we still need an "error info" section at all, I except it will not be in SKContext, but in a new SKResult (name tbd) class that strictly captures execution output. We'll follow up offline.